### ISSUE TITLE
"Browser Plugin Details" permission denied

### ISSUE THREAD

**Issue by justsome12** [sereneblue/chameleon#511]
with C as real profile or disabled:
```
Browser Plugin Details
undefined

Bits of identifying information: 0.76
One in x browsers have this value: 1.69
```

with C as Random Profile Desktop with nothing enabled:
```
Browser Plugin Details
permission denied

Bits of identifying information: 7.94
One in x browsers have this value: 246.37
```

**Comment by sereneblue** [comment]
> with C as real profile or disabled:
> 
> ```
> Browser Plugin Details
> undefined
> 
> Bits of identifying information: 0.76
> One in x browsers have this value: 1.69
> ```
> 
> with C as Random Profile Desktop with nothing enabled:
> 
> ```
> Browser Plugin Details
> permission denied
> 
> Bits of identifying information: 7.94
> One in x browsers have this value: 246.37
> ```

Firefox doesn't use navigator.plugins. Some of the other browser profiles do. It seems the spoofed implementation isn't 100% correct because it's failing for find the existing plugins.

**Comment by sereneblue** [comment]
@justsome12 This is now resolved in v0.22.22.

**Comment by sereneblue** [comment]
@bambirombi That was probably the Internet Explorer browser profile. I did some tests with IE11 in a virtual machine and it looks like this is actually a bug. I've fixed this in v0.22.30 which will be released soon.

**Comment by bambirombi** [comment]
Version: 0.22.31 It works.
